lib/FluxComponent.js

```diff
@@ -3,11 +3,11 @@
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
 var React = require("react");
-var shallowEqual = require("react/lib/shallowEqual");
+var shallowEqual = require("fbjs/lib/shallowEqual");
```

Comment:
I wouldn't put a lot of confidence in using `fbjs` as a dependency, given their warning on their repo:

```
If you are consuming the code here and you are not also a Facebook project, be prepared for a bad time. APIs may appear or disappear and we may not follow semver strictly, though we will do our best to.  [...]
```

[React Pure Render#shallowequal](https://github.com/gaearon/react-pure-render#shallowequal) would probably be a better choice here.